packages/coordinator/src/services/translation/response/cancellation/cancellation-response.ts

```diff
@@ -63,7 +63,7 @@ function createBundleEntries(cancellationResponse: hl7V3.CancellationResponse) {
   //The ResponsibleParty represents the author of the original prescription (if different to the cancel requester)
   const originalPrescriptionAuthorAgentPerson = cancellationResponse.responsibleParty?.AgentPerson
   let translatedOriginalPrescriptionAuthor = translatedCancelRequester
-  if (originalPrescriptionAuthorAgentPerson) {
+  if (originalPrescriptionAuthorAgentPerson.agentPerson) {
```

Comment:
This is now assuming that the responsible party is present which I don't think is always true